2020-02-Spring/2_linked_lists/reorder_list/cpp/Makefile

```diff
@@ -0,0 +1,13 @@
+CXX=g++
+CXXFLAGS = -Wall -ggdb -O0 -std=c++11
+
+all: Solution_Manual Solution_SmartPtr
+
+Solution_Manual: Solution_Manual.o
+	$(CXX) $(CXXFLAGS) -o Solution_Manual Solution_Manual.cpp
+
+Solution_SmartPtr: Solution_SmartPtr.o
+	$(CXX) $(CXXFLAGS) -o Solution_SmartPtr Solution_SmartPtr.cpp
+
+clean:
+	rm -f *.o Solution_Manual Solution_SmartPtr *~
```

Comment:
ooooo fancy! :0

Reply:
This is pretty awesome. (And TBH I have no idea what it means)

Reply:
Me neither, I just copied a template lmao

But now people can compile the c++ solutions without having to spin up an IDE 👍